core/templates/dev/head/profile/Profile.js

```diff
@@ -116,6 +124,7 @@ oppia.controller('Profile', [
       $scope.profilePictureDataUrl = (
         data.profile_picture_data_url || DEFAULT_PROFILE_PICTURE_URL);
       $rootScope.loadingMessage = '';
+
```

Comment:
Yeah that was an accident I made before I actually figured out where to put the code.